main.go

```diff
@@ -523,6 +523,56 @@ func main() {
 			return err
 		}
 
+		//
+		// hc-static-site
+		//
+		// There's enough different about this repo's config that it makes
+		// sense just to start from scratch.
+		hcStaticSiteRepositoryArgs := github.RepositoryArgs{
+			Name:                pulumi.String("hc-static-site"),
+			Description:         pulumi.String("Static website"),
+			Visibility:          pulumi.String("private"),
+			HasDownloads:        pulumi.Bool(true),
+			HasIssues:           pulumi.Bool(true),
+			HasProjects:         pulumi.Bool(true),
+			HasWiki:             pulumi.Bool(false),
+			VulnerabilityAlerts: pulumi.Bool(false),
+			AllowAutoMerge:      pulumi.Bool(false),
+			DeleteBranchOnMerge: pulumi.Bool(false),
+			AllowUpdateBranch:   pulumi.Bool(false),
+			AllowSquashMerge:    pulumi.Bool(false),
+			AllowRebaseMerge:    pulumi.Bool(false),
+			AllowMergeCommit:    pulumi.Bool(false),
```

Comment:
Mmh, if they're all disallowed, I wonder how branches should be merged. Usually we have squash and rebase merges.